app/controllers/students_controller.rb

```diff
@@ -31,6 +27,17 @@ def show
     @past_appointments = @calendar.past_appointment_with_attendee(@appointments)
   end
 
+  def import
+    begin
+      @student = Student.import(params[:user_id], params[:file])
+      flash[:success] = "Students successfully added"
+      redirect_to user_path(@user)
+    rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?